lldb/packages/Python/lldbsuite/test/gdbclientutils.py

```diff
@@ -242,7 +245,7 @@ def qProcessInfo(self):
     def qHostInfo(self):
         return "ptrsize:8;endian:little;"
 
-    def qEcho(self):
+    def qEcho(self, _: int):
```

Comment:
I don't know what this means. Does someone call this with an extra parameter?

Reply:
`_` is an ordinary parameter but this specific name is used as a placeholder for unused values so `_: int` is an integer parameter that's unused. The call that passes an int is on line 191 (https://github.com/llvm/llvm-project/pull/162172/files#diff-f47db48baf9a40e29541b77d2f05362f5074e19fa19d8c30037a75a4c275a517R191) and there are no other calls

Reply:
There is another class that adds the parameter:
```
            def qEcho(self, num):
                resp = "qEcho:" + str(num)
                if num >= 2:
                    # We have launched our program
                    self.started = True
                    return [resp, "T13"]

                return resp
```
For documentation purposes, just call the parameter "num" then. All calls to qEcho( are passing a number so it should be fine to add the parameter here.